lib/tools/apk-utils.js

```diff
@@ -524,6 +524,9 @@ apkUtilsMethods.install = async function install (appPath, options = {}) {
       output : `${output.substr(0, 150)}...${output.substr(output.length - 150)}`;
     log.debug(`Install command stdout: ${truncatedOutput}`);
     if (/\[INSTALL[A-Z_]+FAILED[A-Z_]+\]/.test(output)) {
+      if (/\[INSTALL_FAILED_TEST_ONLY\]/.test(output)) {
+        log.error('Set `allowTestPackages` capability to true in order to allow test packages installation.');
```

Comment:
this is not an error. Also, it makes since to include this hint into the error description above as not all users are checking server logs

Reply:
sounds good to add it in error message

Reply:
@mykola-mokhnach I'll change it to a `warning`.